site/source/service-guides/lightning/fully-noded-lnd.rst

```diff
@@ -13,8 +13,6 @@ Available For:
 #. Download Fully Noded from the Apple App Store.
 #. Log into your Start9 server UI and select Services -> LND -> Properties.
 #. Click the QR code icon next to “LND Connect REST URL” to display the QR code and scan/copy this with your iPhone/mac.
-## [Scanning QR] From the App, you have to go > Node manager > Add a node + > hit Scan QR (not LND)
-## [Pasting credentials] From the App, you have to go > Node manager > Add a node (+) > select LND (not Scan QR). 
```

Comment:
I'm not sure I understand this PR. These two lines seem like pertinent information we want included in the guide rather than comments with the incorrect syntax.

That said, line 16 should maybe just be a single `#` character to include it in the numbered list. Then `[Pasting Credentials]` might appear as a either a separate note or section with the subsequent lines about the 4 fields to be copy/pasted.

Reply:
Right... I was absolutely convinced that these were commented out do to a change and that copying and pasting was the recommended way. It's been malformed for some time (https://docs.start9.com/0.3.5.x/service-guides/lightning/fully-noded-lnd#fully-noded-lnd)

Perhaps the were meant to be tabs at one point? You know, I think that make sense.

Reply:
@Dominion5254 OK check it now, I'm pretty sure these were tabs that were never converted to reStructuredText format.